README.md

```diff
@@ -25,639 +25,7 @@ Before you can integrate with the Ravelin mobile SDK for iOS, you will need:
-
-> Also available with a completion handler: trackPage (pageTitle, eventProperties, completionHandler)
-
-
----
-
-
-## trackLogin (pageTitle, eventProperties)
-
-Sends a track login event to Ravelin
-
-### Parameters
-
-| Parameter           | Type            | Description  |
-| --------------------|-----------------|----------------------------|
-| pageTitle           | String          | The title of the current page |
-| eventProperties     | Dictionary      | A dictionary of meta data to send with the event |
-
-> Also available with a completion handler: trackLogin (pageTitle, eventProperties, completionHandler)
-
-
----
-
-## trackLogout (pageTitle, eventProperties)
-
-Ends current Ravelin session and sends logout event to Ravelin
-
-### Parameters
-
-| Parameter           | Type            | Description  |
-| --------------------|-----------------|----------------------------|
-| pageTitle           | String          | The title of the current page |
-| eventProperties     | Dictionary      | A dictionary of meta data to send with the event |
-
-> Also available with a completion handler: trackLogout (pageTitle, eventProperties, completionHandler)
-
----
-
-## Ravelin Class Properties
-
----
-
-## apiKey
-
-> The public api key from your dashboard 
-
-
-## customerId
-
-> Your chosen customer id 
-
-
-## tempCustomerId
-
-> Temp customer id  
-
-
-## sessionId (read only)
-
-> The Ravelin generated sessionId
-
-
-## deviceId (read only)
-
-> The Ravelin generated device id 
-
-
-## orderId
-
-> Your chosen order id
-
-# RVNEncryption Class Reference
-
-Note: The RVNEncryption class requires the RavelinEncrypt pod.
-
-## RVNEncryption Class Methods
-
----
-
-## sharedInstance
-
-Get the instantiated RVNEncryption singleton
-
-
-### Return value
-
-> The singleton instance of the class
-
----
-
-## encrypt (pan, month, year, nameOnCard, &error)
-
-Generates encryption payload ready for sending to Ravelin
-
-### Parameters
-
-| Parameter     | Type               | Description  |
-| ------------- |---------------------|-------|
-| pan     | String     | A string representation of the long card number |
-| month     | String     | Expiry month of card (1-12) |
-| year     | String     | Expiry year (2 or 4 digit) |
-| nameOnCard     | String     | The customer name on the card |
-| error     | Object     | Passed as reference |
-
-### Return value
-
-> Dictionary containing methodType, aesKeyCiphertext, cardCiphertext, algorithm, keyIndex and ravelinSDKVersion
-
---- 
-
-## RVNEncryption Class Properties
-
----
-
-## rsaKey
-
-> The public RSA key from your dashboard
-
-
-# License
-
-License information can be found [here](LICENSE)
+* [Further information on using RavelinCore](https://github.com/unravelin/ravelin-ios/blob/0.3.0-docs/RavelinCore.md)
+* [Further information on using RavelinEncrypt](https://github.com/unravelin/ravelin-ios/blob/0.3.0-docs/RavelinEncrypt.md)
```

Comment:
Should we add a new line here with a link to our developer docs??

for more details check [Ravelin Documentation for iOS SDK] (https://developer.ravelin.com/libraries-and-sdks/mobile-sdk/ios/)